src/mmw/js/src/core/views.js

```diff
@@ -616,17 +615,36 @@ var MapView = Marionette.ItemView.extend({
             additionalShapes = this.model.get('areaOfInterestAdditionals');
 
         if (additionalShapes) {
-            var pointsLayer = new L.GeoJSON(additionalShapes, {
-                onEachFeature: function(feature, layer) {
-                    var message = "Outlet point snapped to nearest stream";
-                    if (feature.properties && feature.properties.original) {
-                        message = "Original clicked outlet point";
-                    }
-
-                    layer.bindPopup(message);
+            var createCustomMarker = function(geoJSONpoint) {
+                var customIconSize = [16,16];
+                var originalPointRingIcon = L.divIcon({
+                    className: 'marker-rwd marker-rwd-1',
+                    iconSize: customIconSize
+                });
+                var nearestStreamRingIcon = L.divIcon({
+                    className: 'marker-rwd marker-rwd-2',
+                    iconSize: customIconSize
+                });
+                var customIcon = nearestStreamRingIcon;
+                var message = "Outlet point snapped to nearest stream";
+                if (geoJSONpoint.properties.original) {
```

Comment:
Consider creating another function which returns the correct icon and message for a given point.